### PR TITLE
SW-5117 Redux and service functions to search/download participant projects list

### PR DESCRIPTION
**Pull request by karthikbtf**
- created service functions and redux
- assumed data model for now, did not merge with existing data model

### CODE REVIEW

File: src/services/ParticipantProjectService.ts [hunk 1]
```diff
@@ -60,6 +66,78 @@ const get = async (participantProjectId: number): Promise<Response2<ParticipantP
...
 
+const downloadList = async (search?: SearchNodePayload, sortOrder?: SearchSortOrder): Promise<string | null> => {
+  return (
+    'Project Name,Participant,Cohort,Phase,Country,Region,Restorable Land,Land Use Model Type\r' +
+    'Andromeda,Cartwheel,Cohort 1,Phase 0 - Due Diligence,Ghana,Sub-Saharan Africa,500,Native Forest'
+  );
+};
+
+const list = async (
+  // TODO: remove locale if we are using BE search API
```
Comment (tommylau523) [inline src/services/ParticipantProjectService.ts:77]: For this to use BE search API we need to add CohortsSearchTable and ParticipantsSearchTable on the BE.

File: src/types/ParticipantProject.ts [hunk 2]
```diff
@@ -30,3 +31,16 @@ export type ParticipantProject = {
   votingDecision: VoteOption;
   whatNeedsToBeTrue: string;
 };
+
+export type ParticipantProjectSearchResult = {
+  cohortId: number;
+  cohortName: string;
+  country: string;
+  id: number;
+  landUseModelType: string;
```
Comment (tommylau523) [inline src/types/ParticipantProject.ts:40]: LandUseModel type is multi-select so string[]

**Review by tommylau523 — APPROVED** [review]

File: src/types/ParticipantProject.ts (hunk 2 above)
Comment (karthikbtf) [reply src/types/ParticipantProject.ts:40]: Thanks for the info, yes we can change this later as needed.
Can a participant project have more than one value for its land use type? If so, we need to change how it is rendered on the table.

Comment (tommylau523) [reply src/types/ParticipantProject.ts:40]: Yes it is a multi-select in the requirements: 

-Native Forest
-Monoculture
-Sustainable Timber
-Other Timber
-Mangroves
-Agroforestry
-Silvopasture
-Other Land-Use Model

Figma designs still need to be updated to reflect this I think.

Comment (karthikbtf) [reply src/types/ParticipantProject.ts:40]: Got it, thanks @tommylau523 ... I can update the table later once the BE types are available.

**Pull request merged**